flink-table/flink-table-planner-blink/src/main/java/org/apache/flink/table/expressions/RexNodeConverter.java

```diff
@@ -344,4 +875,125 @@ private RexNode visitResolvedDistinctKeyReference(ResolvedDistinctKeyReference r
+					List<RexNode> child = convertCallChildren(call.getChildren());
+					SqlFunction sqlFunction = UserDefinedFunctionUtils.createScalarSqlFunction(
+							scalaFunc.functionIdentifier(),
+							scalaFunc.toString(),
+							scalaFunc,
+							typeFactory);
+					return relBuilder.call(sqlFunction, child);
+				} else {
+					FunctionDefinition def = call.getFunctionDefinition();
+					if (conversionsOfBuiltInFunc.containsKey(def)) {
+						RexNodeConversion conversion = conversionsOfBuiltInFunc.get(def);
+						return conversion.convert(call);
+					} else {
+						throw new UnsupportedOperationException(def.toString());
+					}
+				}
+
+			default:
+				throw new UnsupportedOperationException();
+		}
+	}
+
+	private RexNode createCollation(RexNode node, RelFieldCollation.Direction direction,
+			RelFieldCollation.NullDirection nullDirection, Set<SqlKind> kinds) {
+		switch (node.getKind()) {
+			case DESCENDING:
+				kinds.add(node.getKind());
+				return createCollation(((RexCall) node).getOperands().get(0), RelFieldCollation.Direction.DESCENDING,
+						nullDirection, kinds);
+			case NULLS_FIRST:
+				kinds.add(node.getKind());
+				return createCollation(((RexCall) node).getOperands().get(0), direction,
+						RelFieldCollation.NullDirection.FIRST, kinds);
+			case NULLS_LAST:
+				kinds.add(node.getKind());
+				return createCollation(((RexCall) node).getOperands().get(0), direction,
+						RelFieldCollation.NullDirection.LAST, kinds);
+			default:
+				if (nullDirection == null) {
+					// Set the null direction if not specified.
+					// Consistent with HIVE/SPARK/MYSQL/BLINK-RUNTIME.
+					if (FlinkPlannerImpl.defaultNullCollation()
+							.last(direction.equals(RelFieldCollation.Direction.DESCENDING))) {
+						kinds.add(SqlKind.NULLS_LAST);
+					} else {
+						kinds.add(SqlKind.NULLS_FIRST);
+					}
+				}
+				return node;
+		}
+	}
+
+	private RexWindowBound createBound(Expression bound, SqlKind sqlKind) {
+		if (bound instanceof CallExpression) {
+			CallExpression callExpr = (CallExpression) bound;
+			FunctionDefinition func = callExpr.getFunctionDefinition();
+			if (BuiltInFunctionDefinitions.UNBOUNDED_ROW.equals(func) || BuiltInFunctionDefinitions.UNBOUNDED_RANGE
+					.equals(func)) {
+				SqlNode unbounded = sqlKind.equals(SqlKind.PRECEDING) ? SqlWindow
+						.createUnboundedPreceding(SqlParserPos.ZERO) :
+						SqlWindow.createUnboundedFollowing(SqlParserPos.ZERO);
+				return RexWindowBound.create(unbounded, null);
+			} else if (BuiltInFunctionDefinitions.CURRENT_ROW.equals(func) || BuiltInFunctionDefinitions.CURRENT_RANGE
+					.equals(func)) {
+				SqlNode currentRow = SqlWindow.createCurrentRow(SqlParserPos.ZERO);
+				return RexWindowBound.create(currentRow, null);
+			} else {
+				throw new IllegalArgumentException("Unexpected expression: " + bound);
+			}
+		} else if (bound instanceof ValueLiteralExpression) {
+			int DECIMAL_PRECISION_NEEDED_FOR_LONG = 19;
```

Comment:
code style error..

Reply:
sorry for that, @JingsongLi fixed the compile error in https://github.com/apache/flink/pull/8689

Reply:
When I merging https://github.com/apache/flink/pull/8979, I found the CI failed,
And I want try to hotfix it, but after rename the `ConvertGet` -> `convertGet`, rebuild again, then another check style error threw. So I am not pretty sure which way is better way to fix it.

Reply:
@sunjincheng121 Those two compile error in this pr already fix in https://github.com/apache/flink/pull/8689, which is merged already.  Could you try again ?
I'm sorry for the inconvenience.